packages/cli/package.json

```diff
@@ -70,7 +70,9 @@
     "path": "^0.12.7",
     "solmate": "https://github.com/Rari-Capital/solmate.git#9cf1428245074e39090dceacb0c28b1f684f584c",
     "table": "^6.8.1",
+    "ts-node": "^10.9.1",
```

Comment:
why is ts-node a dependency instead of a dev dependency now?

Reply:
shebang relies on it. It could be a peerDep, but bad devex. Ultimately it's needed to load the ts config file